### ISSUE TITLE
Broken upstream git repo?

### ISSUE THREAD

**Issue by vchernin** [flathub/org.freedesktop.LinuxAudio.Plugins.ZamPlugins#4]
This is surely not this repo's fault, but the current `branch/21.08` is not buildable locally for me (and I assume buildbot will be the same).

While trying to update pulseeffects (which historically has its own bundled zamplugins), flatpak-builder fails to download the zamplugins git repo. Zamplugins also fails to download when building this repo.
```
Initialized empty Git repository in /var/home/v/devel/org.freedesktop.LinuxAudio.Plugins.ZamPlugins/.flatpak-builder/git/https_github.com_zamaudio_zam-plugins.git-9X04J1/
Fetching git repo https://github.com/zamaudio/zam-plugins.git, ref refs/tags/3.14
remote: Enumerating objects: 471, done.
remote: Counting objects: 100% (471/471), done.
remote: Compressing objects: 100% (443/443), done.
remote: Total 471 (delta 184), reused 233 (delta 26), pack-reused 0
Receiving objects: 100% (471/471), 15.99 MiB | 10.05 MiB/s, done.
Resolving deltas: 100% (184/184), done.
From https://github.com/zamaudio/zam-plugins
 * [new tag]         3.14       -> 3.14
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in /var/home/v/devel/org.freedesktop.LinuxAudio.Plugins.ZamPlugins/.flatpak-builder/git/git_github.com_DISTRHO_DPF-FBZ8J1/
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
Failed to download sources: module zam-plugins: Child process exited with code 128
```

I get the error on Fedora 36 Silverblue using Fedora's flatpak-builder and org.flatpak.Builder.

I don't understand why zamplugins alone causes this problem, why it's seemingly treated differently by flatpak-builder.

**Comment by hfiguiere** [comment]
It's kinda GitHub fault.

https://github.blog/2021-09-01-improving-git-protocol-security-github/

The only solution I see is disabling submodules and adding them as manual source.

**Comment by vchernin** [comment]
Yeah, that works. I also sent a patch https://github.com/zamaudio/zam-plugins/pull/85

```
- type: git
  disable-submodules: true
  url: 'https://github.com/zamaudio/zam-plugins'
  commit: e7077fcc0b7f60daa7471eae42015ffc9cba73d9
  tag: '3.14'
  x-checker-data:
    type: json
    url: https://api.github.com/repos/zamaudio/zam-plugins/releases/latest
    tag-query: .tag_name
    version-query: .tag_name
    timestamp-query: .published_at
    is-main-source: true
- type: git
  url: https://github.com/DISTRHO/DPF.git
  commit: 08669d1bc30c6e971fde800eade4ca40104ba8b2 # as used in https://github.com/zamaudio/zam-plugins/tree/3.14
  dest: dpf
```

**Comment by vchernin** [comment]
Fixed in upstream commit `3f751bed7dfa1e43f7b3bf26f8368eb55c040a55`
